frontend/src/domains/hooks/useCheerButton.ts

```diff
@@ -11,12 +12,31 @@ export default function useCheerButton({
 }: UseCheerButtonProps) {
   const [count, setCount] = useState(0);
   const [animate, setAnimate] = useState(false);
+  const [isDisabled, setIsDisabled] = useState(false);
+  const { showToast } = useToast();
 
   const handleCheerButton = () => {
+    if (isDisabled) {
+      return;
+    }
     setCount(count + 1);
     setAnimate(false);
     requestAnimationFrame(() => setAnimate(true));
-    debouncedSearch(count + 1);
+
+    const newCount = count + 1;
+
+    if (newCount === 100) {
```

Comment:
```suggestion
    if (newCount >= 100) {
```

요렇게 이상으로 조건 수정하는게 조금 더 안전할 것 같아요~

Reply:
좋습니다! 그렇게 수정했습니다~!